### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by chalin** [wjdp/htmltest#174] [General]
There have been some nice changes since 0.14.0 was released in January. Will a new release be cut soon? Thanks!

**Comment by wjdp** [comment]
Yes, will try to do this next week.

(For context I'll run the current master against a few projects to validate then make the tag)

**Reply by chalin** [reply]
> (For context I'll run the current master against a few projects to validate then make the tag)

We've been running `htmltest`@HEAD on the following sites for a while now:

- [etcd.io](https://etcd.io/): 
  see, https://github.com/etcd-io/website/blob/10e2e82a725693c0e4387312891788d741daf996/Makefile#L22-L32
- [grpc.io](https://grpc.io/): 
  see, https://github.com/etcd-io/website/blob/10e2e82a725693c0e4387312891788d741daf996/Makefile#L22-L32
- [notaryproject.dev](https://notaryproject.dev/): 
  see, https://github.com/notaryproject/notaryproject.dev/blob/82774640a1f1447370da43f13939745862775bba/Makefile#L23-L32

Hopefully this can help validate that `htmltest`@HEAD is "production ready".
Is there anything I can do to help get the next release out?

Btw, it would be great if #175 could be merged before the next release :).

**Comment by chalin** [comment]
Hi, any updates or new ETA for the next release?

**Comment by wjdp** [comment]
Sorry for the delay, now out https://github.com/wjdp/htmltest/releases/tag/v0.15.0

Did this before checking #175 as didn't want to delay anymore but will look into this now.

**Reply by chalin** [reply]
Awesome on both counts! 😄 Thanks.

**Comment by chalin** [comment]
Thanks for your recent activity @wjdp. Do you have an idea when you'll be putting out the next release?

**Comment by wjdp** [comment]
Have made a RC release https://github.com/wjdp/htmltest/releases/tag/v0.16.0-rc2

Will be making a proper one once the apple arm users have been given a chance to test.

**Reply by wjdp** [reply]
@chalin Release made https://github.com/wjdp/htmltest/releases/tag/v0.16.0

**Reply by chalin** [reply]
Thanks!

**Comment by kylrth** [comment]
Hey, how about another one? I'd really like to use the `IgnoreHTTPS` option in my CI.

**Reply by wjdp** [reply]
Sorry for the (very) long delay, https://github.com/wjdp/htmltest/releases/tag/v0.17.0 includes this

**Comment by istr** [comment]
Hello @wjdp,

Do you see any chance to ship v0.18.0?

I would love to see https://github.com/wjdp/htmltest/commit/c9077f50021eb45f8f0364a7c84511640024f0dc (ignore prefetch/preconnect) in it.
